docs: improvements on language from cluster peering steps (#14993)
hashicorp/consul: website/content/docs/connect/cluster-peering/k8s.mdx

The examples described in this section demonstrate how to export a service named `backend`. You should change instances of `backend` in the example code to the name of the service you want to export.

> 1. For the service in `cluster-02` that you want to deploy onto the mesh and evautally export, ensure that you add the `"consul.hashicorp.com/connect-inject": "true"` annotation to your service's pods prior to deploying. Below is an example workload with the annotation highlighted that allows the workload to join the mesh. 

Reviewer: ```suggestion
1. For the service in `cluster-02` that you want to export, add the `"consul.hashicorp.com/connect-inject": "true"` annotation to your service's pods prior to deploying. The annotation allows the workload to join the mesh. It is highlighted in the following example: 
```
I think we might be packing too much workflow detail into this step. Could we add this to the introduction paragraph?

    $ kubectl --context $CLUSTER2_CONTEXT apply --filename intention.yml
    ```

> 1. For the services in `cluster-01` that you want like to deploy and dial `backend` in `cluster-02`, ensure that you add the `"consul.hashicorp.com/connect-inject": "true"` annotation to your service's pods before deploying the workload. To dial the upstream service from an application, ensure that the requests are sent to the correct DNS name as specified in [Service Virtual IP Lookups](/docs/discovery/dns#service-virtual-ip-lookups). Below is an example workload with the annotation highlighted that allows the workload to join the mesh, and config provided to the workload that enables the workload to dial the upstream service using the correct DNS name. 

Reviewer: ```suggestion
1. Add the `"consul.hashicorp.com/connect-inject": "true"` annotation to your service's pods before deploying the workload so that the services in `cluster-01` can dial `backend` in `cluster-02`. To dial the upstream service from an application, verify that requests are sent to the correct DNS name as specified in [Service Virtual IP Lookups](/docs/discovery/dns#service-virtual-ip-lookups). In the following example, the annotation that allows the workload to join the mesh and the configuration provided to the workload that enables the workload to dial the upstream service using the correct DNS name is highlighted. 
```
It feels like a lot fo information for one step. Can you extract any of it as summary information and add it to the introduction to the steps? It seems that (as with the other section) that the core of the example is to add the annotation to the pod configurations you want to change the behavior of. Maybe move the explanations of what the annotations do to the intro?
